### ISSUE TITLE
There is no UI to toggle cardset bottoms

### ISSUE THREAD

**Issue by denilsonsa** [shlomif/PySolFC#347]
4c2dd59928e493b1e552eede7c2cd00a132591d0 added a new option `use_cardset_bottoms`, which toggles between using auto-generated card bottoms, or using the cardset-provided bottoms. That option exists in the config file and it works.

However, so far there is no UI to toggle that option. I expected to have a menu toggle under *Options → Card view*.

---

Well, it works but it can be improved. If the cardset bottom is not found, PySol should probably print a warning message to the terminal (so people would be aware of the issue), and should probably fall back to generating one programmatically (or to using the (non-)negative version).

**Comment by joeraz** [comment]
Agreed.  I'd like to add such an option.  The main issue is that the generated bottoms are loaded separately from the cardset-specific bottoms.  So if I add such an option, it would only work for cardsets loaded after the change.  Would need to alter cardset loading logic to account for this, so it's a little more difficult to do than it sounds.

I've already added the missing bottoms files to the cardsets repo for the next release.
